src/noRecords/NoRecords.styles.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+const StyledEmptyMessage = styled.div`
+  text-align: center;
+  padding: 40px 20px;
+  h2 {
```

Comment:
Instead of targeting the h2 here we should do StyledHeading in the main component so its directly targeting it. I think it was like this in admin because it was built before Heading existed in admin-ui.